### PR TITLE
Project setup

### PR DESCRIPTION
**Pull request by jupabass89**
## Summary
-The file .gitignore with the respective files to be ignored is added to the repository, the file index.html and the folder scs according to the established card setup of the project 

## Screenshot
Evidence

## Trello Card
https://trello.com/c/L51BC8pO/31-pull-request

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,10 @@
+### Sass ###
+.sass-cache/
+.scss-cache/
+*.css.map
+*.sass.map
+*.scss.map
+*.css
+
+## System ##
+.DS_Store
```
Comment (henryzarza) [inline .gitignore]: Add end of file (EOF)

File: index.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  <meta charset="utf-8">
+  <title>HTML & CSS Training</title>
+  <meta name="description" content="training">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet"  type="text/css" href="application.css">
+</head>
+<body>
+  <h1 class="title">Hello World!</h1> 
+</body>
+</html>
```
Comment (henryzarza) [inline index.html]: Add EOF

File: scss/application.scss [hunk 3]
```diff
@@ -0,0 +1,3 @@
+.title {
+  color: red;
+}
```
Comment (henryzarza) [inline scss/application.scss]: The same here

File: .gitignore (hunk 1 above)
Comment (jupabass89) [reply .gitignore]: Te refieres a una linea limpia al final?

Comment (jupabass89) [reply .gitignore]: dou you mean, the last cleean line at the end?

**Pull request merged**